server/src/repositories/search.repository.ts

```diff
@@ -293,6 +293,12 @@ export class SearchRepository {
     });
   }
 
+  async getEmbedding(assetId: string) {
+    return this.db.transaction().execute(async (trx) => {
+      return trx.selectFrom('smart_search').selectAll().where('assetId', '=', assetId).executeTakeFirst();
+    });
+  }
+
```

Comment:
this can just be `return this.db.selectFrom('smart_search' ...)` and you can add a `@GenerateSql({ params: [DummyValue.UUID] })` thing as well.

Reply:
Got it, no `set local ...`, no need for the transaction

And added the `@GenerateSql`